src/typeclass/Semigroup.ts

```diff
@@ -151,6 +151,28 @@ export const booleanAny: Semigroup<boolean> = make(
   }
 )
 
+/**
+ * `boolean` semigroup under exclusive disjunction.
+ *
+ * @category instances
+ * @since 1.0.0
+ */
+export const booleanXor: Semigroup<boolean> = make(
+  (self, that) => self !== that,
+  (self, collection) => {
```

Comment:
Is the implementation of `combineMany` correct? The following test should pass

```ts
deepStrictEqual(Boolean.SemigroupXor.combineMany(true, [false, true]), false)
```

but with the current implementation fails. 

I'm not sure `combineMany` can be optimised, I would expect a definition like

```ts
export const booleanXor: Semigroup<boolean> = make((self, that) => self !== that)
```

Reply:
Agreed, I misunderstood the semantic of `combineMany`. Now it's clear. I will fix it.